astropy/samp/tests/test_web_profile.py

```diff
@@ -68,10 +67,14 @@ def test_web_profile(self):
 
         # Check some additional queries to the server
 
-        with get_readable_fileobj(f'http://localhost:{self.hub._web_port}/crossdomain.xml') as f:
+        with get_readable_fileobj(
+            f'http://localhost:{self.hub._web_port}/crossdomain.xml'
+        ) as f:
             assert f.read() == CROSS_DOMAIN
 
-        with get_readable_fileobj(f'http://localhost:{self.hub._web_port}/clientaccesspolicy.xml') as f:  # noqa: E501
+        with get_readable_fileobj(
+            f'http://localhost:{self.hub._web_port}/clientaccesspolicy.xml'
+        ) as f:  # noqa: E501
```

Comment:
Wonder if we can remove the noqa?

Reply:
This maybe stripped by pre-commit once we normalize the strings.

Reply:
Looks like it was fixed!